config/json/db.json

```diff
@@ -4,10 +4,19 @@
       "url": "/not-found",
       "state": "notFound",
       "template": "not-found",
-      "endpoint": "post",
-      "params": {
-        "id": 1
-      }
+      "request": false
+    },
+    {
+      "url": "/patterns",
```

Comment:
I think is better this way, because each app can point to a different not-found template. In our case all apps have that template, but maybe other 3rd party apps have a template with a different name.